static-assets/scripts/main.js

```diff
@@ -758,7 +758,7 @@
             }
 
             function authLoop() {
-                $scope.authDelay = 1000;
+                $scope.authDelay = 60000;
```

Comment:
What's this about?

Reply:
This is a timer for token validation, set to the same time as yui code, should have been a separate commit